.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,109 @@
+name: Zeebe CI (AWS OS)
```

Comment:
I'd use a different name since the tests this workflow is running are not all related to Zeebe. Maybe just "AWS OpenSearch Dispatch Tests" orso.